source/postprocess/boundary_velocity_residual_statistics.cc

```diff
@@ -221,89 +225,49 @@ namespace aspect
 
       // now add the computed max, min, and rms velocities to the statistics object
       // and create a single string that can be output to the screen
+      const std::string units = (this->convert_output_to_years() == true) ? " (m/yr)" : " (m/s)";
+      const double unit_scale_factor = (this->convert_output_to_years() == true) ? year_in_seconds : 1.0;
       std::ostringstream screen_text;
       unsigned int index = 0;
+
       for (std::map<types::boundary_id, double>::const_iterator
            p = global_max_vel.begin(), a = global_min_vel.begin(), rms = global_rms_vel.begin();
            p != global_max_vel.end() && a != global_min_vel.end() && rms != global_rms_vel.end();
            ++p, ++a, ++rms, ++index)
         {
-          if (this->convert_output_to_years() == true)
```

Comment:
Do you know what the meaning of p and a is? Can you replace the variable names please?